src/css/components/panel.scss

```diff
@@ -121,3 +121,17 @@ h4 + .panel {
   flex-shrink: 1;
   min-width: 25%;
 }
+
+.panel-new {
```

Comment:
I think it will be — but I wasn't ready to redefine all the panels just now. too many ripples